iidm-converter-api/src/main/java/eu/itesla_project/iidm/import_/Importers.java

```diff
@@ -339,6 +343,10 @@ public static ReadOnlyDataSource createReadOnly(Path directory, String fileNameO
             return new GzFileDataSource(directory, getBaseName(fileNameOrBaseName.substring(0, fileNameOrBaseName.length() - 3)));
         } else if (fileNameOrBaseName.endsWith(".bz2")) {
             return new Bzip2FileDataSource(directory, getBaseName(fileNameOrBaseName.substring(0, fileNameOrBaseName.length() - 4)));
+        } else if (fileNameOrBaseName.endsWith("_EQ.xml")) {
```

Comment:
This has nothing to do here. _EQ is related to CIM data and it totally the plugin architecture of importers